en/physics/physics-item.md

```diff
@@ -27,6 +27,16 @@ If you use __builtin__ for development, please note the following:
 
 It should be noted that currently the __ammo.js__ module has a size of about __1.5MB__.
 
+## Physics Engine: PhysX
+
+**PhysX**（[GitHub](https://github.com/NVIDIAGameWorks/PhysX)）is an open source real-time commercial physics engine developed by Nvidia. It has perfect functional characteristics and extremely high stability, as well as excellent performance.
+
+**PhysX** currently supported in Cocos Creator is version 4.1, which is only allowed to be used on most native platforms. In addition, since PhysX does not support the Android x86 platform, it cannot run normally on this platform. At the same time, some newer platform devices, such as Apple M1 chip devices, will be supported in the future.
```

Comment:
```suggestion
**PhysX** currently supported in Cocos Creator is v4.1, which is only allowed to be used on most native platforms. In addition, since PhysX does not support the Android x86 platform, it cannot run normally on this platform. At the same time, some newer platform devices, such as Apple M1 chip devices, will be supported in the future.
```